test/e2e/go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/vmware-tanzu/tanzu-cli/test/e2e
+
+go 1.19
+
+replace github.com/vmware-tanzu/tanzu-cli => ../..
+
+require (
+	github.com/onsi/ginkgo v1.16.5
```

Comment:
@chandrareddyp ginkgo version is upgraded to  v2   in main. pls rebase and use ginkgo v2 here